projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -103,19 +101,29 @@ export class NavigationListComponent implements OnChanges {
       .getCombinedFeatureState([ApplicationFeature.PageTimeRange, ApplicationFeature.NavigationRedesign])
       .pipe(
         map(featureState => featureState === FeatureState.Enabled),
-        tap(usePageLevelTimeRange => (this.usePageLevelTimeRange = usePageLevelTimeRange)),
         switchMap(usePageLevelTimeRange => {
           if (usePageLevelTimeRange) {
             return this.navListComponentService.resolveNavItemConfigTimeRanges(this.navItems);
           }
 
           return of(this.navItems);
-        })
+        }),
+        shareReplay()
       );
 
-    this.activeItem$ = this.navigationService.navigation$.pipe(
-      startWith(this.navigationService.getCurrentActivatedRoute()),
-      map(() => this.findActiveItem(this.navItems))
+    this.activeItem$ = combineLatest([
+      this.navigationService.navigation$.pipe(startWith(this.navigationService.getCurrentActivatedRoute())),
+      this.navItems$
+    ]).pipe(
+      map(([, navItems], index) => {
+        const activeItem = this.findActiveItem(navItems);
+        console.log('index', index);
+        console.log('active', activeItem);
+        if (index === 0 && activeItem) {
```

Comment:
Have to be careful here, a number of things that can go wrong I believe:
1. Nav list is recreated in certain scenarios I believe, such as going to/from settings.
2. Deep links with a time range should be respected, and take priority over any defaults, user assigned or otherwise.
3. What behavior do we want if the initial load is not for the top level? This behavior seem reasonable, but it's worth calling out this will assign the top level nav item's default if I load up a child page as my initial load. (In other words, if I deep link to endpoint detail, what time range should it get - currently it will inherit from its parent left nav item).

Reply:
I see, yeah good points. So the behaviour we want for case three is correct - if we deeplink to a nested page, we want the time range to inherit from the parent left nav item, because normally that's where it would be manually selected then carried forward to the drill down page. 

For point one, this shouldn't be an issue since it won't be the first navigation for the settings pages correct ? 

For point two, this case is covered in the existing logic of `time-range.service` - on first navigation event, the time range is taken from the query params and set accordingly, so i guess we just need to ensure it is not overwritten in this logic.

Reply:
Just to summarize our offline discussion here:
> this shouldn't be an issue since it won't be the first navigation for the settings pages correct

We just need to make sure if we return from settings, we maintain the time range we arrived with (assuming we used our app back functionality)

> index will be > 0

In general, index is not trust worthy, since that index is local to the observable created in the component, which itself can be recreated.

> this case is covered in the existing logic of time-range.service

The big gist of our discussion is we don't want two different places in the app (left nav and TR service), both fighting over setting the defaults. If we want it set based on page, remove the default from TR service. There's concern over timing here, since TR service initializes earlier than left nav. That is fair, but actually highlights the importance here - we don't want any code reading one default and firing a query based on that, when the loading will eventually settle on a different PR. We need to flush out any such issues and make sure they're waiting for the TR to be resolved.

Either way, we need to respect deep link time ranges though, so if I'm deep linked to an explorer screen with a TR, it should come from query param. If I'm deep linked to an explorer screen without a TR, it should be determined by one place (which looks like the left nav component in current PR).